test/utils/createMount.tsx

```diff
@@ -85,6 +85,7 @@ export default function createMount(options: CreateMountOptions = {}) {
 
   afterEach(() => {
     ReactDOMTestUtils.act(() => {
+      // eslint-disable-next-line react/no-deprecated
```

Comment:
I couldn't make it work. This could be caused by the lack of Enzyme support for React 18.